api/v1beta1/typeoverrides.go

```diff
@@ -475,6 +475,9 @@ type RouteOpenShiftV1Spec struct {
 
 	TLS *OpenshiftTLSConfig `json:"tls,omitempty" protobuf:"bytes,6,opt,name=tls"`
 
+	// +optional
+	Subdomain *string `json:"subdomain,omitempty" protobuf:"bytes,8,opt,name=subdomain"`
```

Comment:
In the upstream library, it's not a pointer, so I guess we should switch to `string` here as well. In the end, an empty string is not ambiguous in this context.
```suggestion
	Subdomain string `json:"subdomain,omitempty" protobuf:"bytes,8,opt,name=subdomain"`
```

Reply:
Weird, I thought I pasted that in, must be a leftover from something else, good catch!